theme/templates/article.html

```diff
@@ -104,6 +104,8 @@ <h2 class="top-subheader">{{ article.headline }}</h2>
         {{ article.content }}
     </article>
 
+    <div class="addthis_inline_share_toolbox"></div>
```

Comment:
Move this inside the `if ADDTHIS_PUBID` statement and place the entire block above the comment section.

Also make sure that the social sharing buttons looks like the rest of the page